### DISCUSSION TITLE
Vite+React js Multiple webpage site deployment issue

### DISCUSSION THREAD

**Discussion by GehanPasindhu** [vitejs/vite-plugin-react#162] [Q&A]
I have a vite+react.js multi-page website. It can easily access the other pages developed as jsx components in development. As an example, if I try to access localhost/about-us it works perfectly. But when it comes to the build version and deploying it to the production, I can't access the same page (domainname.come/about-us). it returns a 404 error. How to fix this error

**Comment by ArnaudBarre** [comment]
You should configure your hosting provider to returns `/` instead of 404 when the route doesn't look like an extension (you can search for "SPA" mode)

**Reply by GehanPasindhu** [reply]
No I want to show the about us component in the file url when it opens the url. Like simply in netlify we using redirects file to redirect files to index.html and it loads other pages. I want to do the same with lightsail

**Reply by ArnaudBarre** [reply]
You will need to look for the documentation of the hosting provider, I can't really help you here 🤷‍♂️
